src/console_reporter.cc

```diff
@@ -45,19 +46,26 @@ bool ConsoleReporter::ReportContext(const Context& context) {
     GetErrorStream()
         << "Color printing is only supported for stdout on windows."
            " Disabling color printing\n";
-    output_options_ = static_cast< OutputOptions >(output_options_ & ~OO_Color);
+    output_options_ = static_cast<OutputOptions>(output_options_ & ~OO_Color);
   }
 #endif
 
   return true;
 }
 
 void ConsoleReporter::PrintHeader(const Run& run) {
-  std::string str = FormatString("%-*s %13s %13s %10s", static_cast<int>(name_field_width_),
-                                 "Benchmark", "Time", "CPU", "Iterations");
-  if(!run.counters.empty()) {
-    if(output_options_ & OO_Tabular) {
-      for(auto const& c : run.counters) {
+  std::string str =
+      (output_options_ & OO_Base_Name)
```

Comment:
It's not strictly necessary, but I wanted to leave this as a way for someone to (easily) enable printing the base name for the console reporter, maybe gated by an option or something. I didn't add any such option here, it's just in case somebody wants it.

Reply:
We should either make it fully functional with an option, or remove it. We can always create an issue as a "good first issue" to add this support and option as a followup. 

that'll make it easier to understand what was added in this PR.

Reply:
I removed the code for it: I suppose somebody else can implement the option for console reporting and the like!